Gem/Swap/Scenes/SwapScene.swift

```diff
@@ -16,10 +16,11 @@ import InfoSheet
 import Gemstone
 
 struct SwapScene: View {
+    static let maxBps = 10_000
```

Comment:
```suggestion
    static let maxBps = 10_000
```

what is the purpose and why is it in SwapScene?